src/common/java/net/minecraftforge/gradle/common/util/RunConfig.java

```diff
@@ -68,6 +69,7 @@ public class RunConfig extends GroovyObjectSupport implements Serializable {
     private Boolean client; // so we can have it null
     private Boolean inheritArgs;
     private Boolean inheritJvmArgs;
+    private final Property<String> folderName;
```

Comment:
This is not gonna work. RunConfigs are serializable objects. using properties here is a bit useless and should be avoided.